frontend/model/contracts/chatroom.js

```diff
@@ -129,26 +130,7 @@ sbp('chelonia/defineContract', {
       }
     }
   },
-  getters: {
-    currentChatRoomState (state) {
-      return state
-    },
-    chatRoomSettings (state, getters) {
-      return getters.currentChatRoomState.settings || {}
-    },
-    chatRoomAttributes (state, getters) {
-      return getters.currentChatRoomState.attributes || {}
-    },
-    chatRoomMembers (state, getters) {
-      return getters.currentChatRoomState.members || {}
-    },
-    chatRoomRecentMessages (state, getters) {
-      return getters.currentChatRoomState.messages || []
-    },
-    chatRoomPinnedMessages (state, getters) {
-      return (getters.currentChatRoomState.pinnedMessages || []).sort((a, b) => a.height < b.height ? 1 : -1)
-    }
-  },
+  getters: {},
```

Comment:
Why were these getters removed?

Reply:
Because they've been moved to the chatroom module.

Reply:
I see that, but that doesn't explain why that was done.